src/thumbnailer.h

```diff
@@ -93,14 +93,14 @@ class Thumbnailer {
   // GenerateAnimationEqualQuality() or GenerateAnimationEqualPSNR() must be
   // called before to generate animation with lossy encoding. In case of
   // failure, returns the latest lossy encoded frames.
-  Status NearLosslessDiffPreProcessing(WebPData* const webp_data);
+  Status NearLosslessDiffPrediction(WebPData* const webp_data);
```

Comment:
I do not get why it is called "Prediction"; what is being predicted from what context?